File: app/src/main/java/antoninovitale/dropcodechallenge/api/model/Method.java
package antoninovitale.dropcodechallenge.api.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class Method implements Serializable {
    @SerializedName("mash_temp")
    @Expose
    private List<MashTemp> mashTemp;

    @SerializedName("fermentation")
    @Expose
    private Fermentation fermentation;

    public List<MashTemp> getMashTemp() {
        return mashTemp;
    }

    public void setMashTemp(List<MashTemp> mashTemp) {
        this.mashTemp = mashTemp;
    }

    public Fermentation getFermentation() {
        return fermentation;
    }

    public void setFermentation(Fermentation fermentation) {
        this.fermentation = fermentation;
    }

}

File: app/src/main/java/antoninovitale/punkapi/app/api/model/Method.kt
package antoninovitale.punkapi.app.api.model

import com.squareup.moshi.Json
import se.ansman.kotshi.JsonSerializable
import java.io.Serializable

@JsonSerializable
data class Method(
        @Json(name = "mash_temp") var mashTemp: List<MashTemp>? = null,
        var fermentation: Fermentation? = null
) : Serializable